### PR TITLE
Lawnfeed updater

### PR DESCRIPTION
**Pull request by divadsn**

### CODE REVIEW

**Comment by divadsn** [comment]
It only requires to add a updater task into Lawnfeed on init to check the version.json and show a notification if a new update is available.

File: app/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ def getChangelog = { ->
     if (commitRange) {
         if (System.getenv("TRAVIS_TAG")) {
             def tag = System.getenv("TRAVIS_TAG")
-            commitRange = ('git describe --abbrev=0 --tags ' +  tag + '^').execute().text.trim() + '...' + tag
```
Comment (fonix232) [inline app/build.gradle:9]: Made the same mistake :joy:

**Comment by divadsn** [comment]
Codacy derped again xD

**Comment by LinuxLlama** [comment]
@divadsn Will this give a download link to the latest version of Lawnfeed too?

**Comment by divadsn** [comment]
Check https://lawnchair.info/getlawnfeed.html and the previous PRs.

**Comment by divadsn** [comment]
We might probably need to create a fake dummy activity for the sake of the new permissions.

**Comment by divadsn** [comment]
Aaaand the xml vector needs to be fixed, wew.

**Review by fonix232 — APPROVED** [review]

**Comment by fonix232** [comment]
Looks good to me

**Review by msfjarvis — APPROVED** [review]
LGTM. I'm bad with Cursors so Jozsef or Till can give it a second glance.

**Review by suphon-t — APPROVED** [review]
Looks good.

**Pull request merged**